scripts/create-i18n.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const path = require('path');
+
+const cwd = process.cwd();
+
+// Generate i18n.import.ts
+const i18nFile = path.join(cwd, 'src', '_locales', 'en', 'messages.json');
+
+const messages = require(i18nFile);
+const filename = path.join(cwd, 'src', 'i18n.import.ts');
+const content = `// autogenerated by scripts/create-i18n.js
+import { getMessage as getMessageGlobal, GetMessageOptions } from '@hint/utils/dist/src/i18n/get-message';
+
+export type MessageName =
+${
+    Object.keys(messages).map((name) => {
+        return `    '${name}'`;
+    })
+        .join(' |\n')
+};
+
+export const getMessage = (message: MessageName, options?: GetMessageOptions) => {
+    return getMessageGlobal(message, options);
```

Comment:
I had another idea for allowing us to avoid specifying prefixes on every key. Since we're generating this method now, we could inject the package name here and leave it out of both the source `messages.json` file and callers of this method.

Reply:
That should also make `validate-i18n.js` unnecessary.

Reply:
@antross to get the message, we also pass the language, so basically, for each report we are doing:

```js
context.report('resource', getMessage('MessageKey', {language: context.language, substitutions}, reportOptions));
```

I think it can be easy to forget to pass the language, I can think in three options:
1. Make options y language mandatory
1. Make language another parameter in `getMessage`
1. Make `context` a parameter in `getMessage`.

What do you think?